CMakeLists.txt

```diff
@@ -222,6 +221,11 @@ if(QUDA_MAX_MULTI_RHS_TILE GREATER QUDA_MAX_MULTI_RHS)
   message(SEND_ERROR "QUDA_MAX_MULTI_RHS_TILE is greater than QUDA_MAX_MULTI_RHS")
 endif()
 
+set(QUDA_MAX_KERNEL_ARG_SIZE "4096" CACHE STRING "maximum static size of the kernel arguments in bytes passed to a kernel on the target architecture")
+if(QUDA_MAX_KERNEL_ARG_SIZE GREATER 32768)
```

Comment:
And CUDA max is 32,764 bytes

Reply:
[ea6f025](https://github.com/lattice/quda/pull/1569/commits/ea6f02508d580748cb33db1b132f9410eae659b9)